internal/archive/fsops_test.go

```diff
@@ -111,9 +115,43 @@ var _ = Describe("FSOPS", func() {
 			Entry("Target is empty string",
 				getFullPath("testdata", "mtahtml5"), "", nil, true, nil),
 			Entry("Source is empty string", "", "", nil, true, nil),
+			// folder module (which is symbolic link itself) is archived
+			// it points to folder moduleNew that consists of symlink pointing to content and package.json
+			// etc... thus we check a complex case consisting of normal files/folders and symbolic links that are also
+			// files and folders
+			Entry("symbolic links",
+				getFullPath("testdata", "testsymlink", "module_symlink_dir"), targetFilePath, nil, false,
+				[]string{"module_symlink_dir/", "module_symlink_dir/symlink_dir/", "module_symlink_dir/package.json",
```

Comment:
Should the root be the symlink folder or its content like in the regular (non-symlink) folder case?

Reply:
I didn't understand this comment.